File: 0412-fizz-buzz/0412-fizz-buzz.php
class Solution
{

    /**
     * @param Integer $n
     * @return String[]
     */
    function fizzBuzz($n)
    {
        $output = [];
        for ($i = 1; $i <= $n; $i++) {
            if ($i % 3 == 0 && $i % 5 == 0) {
                $output[] = "FizzBuzz";
            } elseif ($i % 3 == 0) {
                $output[] = "Fizz";

            } elseif ($i % 5 == 0) {
                $output[] = "Buzz";
            } else {
                $output[] = "$i";

            }
        }
    

        return $output;
    }
}

File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        output=[]
        for i in range(n):
            number=i+1
            if number%3 == 0 and number%5 == 0:
                output.append("FizzBuzz")
            elif number % 3==0 :
                 output.append("Fizz")
            elif number % 5==0:
                 output.append("Buzz")
            else:
                output.append(str(number))
        return output